docs/evaluation/how_to_guides/evaluation/llm_as_judge.mdx

```diff
@@ -0,0 +1,83 @@
+import {
+  CodeTabs,
+  python,
+  typescript,
+} from "@site/src/components/InstructionsWithCode";
+
+# How to define an LLM-as-a-judge evaluator
+
+:::info Key concepts
```

Comment:
Shouldn't be part of this PR since it'll take too much time, but users want a lot more guidance on good LLM as judge definition than we give rn. Even our concepts doc isn't very helpful.